changelog.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## What's New (09/12/2017)
+
+### Resolved issues/Bug fixes
+-	Better camelcased suggestions
+-	Better handling of empty operationIds in OpenAPI specs
+-	Case insensitive checks for top level resource properties
+-	Better messaging for tracked resources related rules
+
+### New validation rules
+-	ArraySchemaMustHaveItems – A schema of array type must always contain an items property. without it, AutoRest will fail to generate an SDK. Documentation [link](https://github.com/Azure/azure-rest-api-specs/blob/8ac36944d0ab7f6d07daf077a99bdae873d5e3f6/documentation/openapi-authoring-automated-guidelines.md#L88)
```

Comment:
we may want to indicate here if the new rules are errors or warnings (maybe putting the category in parenthesis), I know it's in the documentation, but it's probably good for someone reading these notes, to know what may affect them right away.